scripts/gcloud.sh

```diff
@@ -289,24 +289,26 @@ gcloud_create_or_update_managed_instance_group() {
   if gcloud compute instance-groups managed describe "${name}" ${gcloud_region} --quiet; then
     # get current instance template name
     local group_instance_name="$(gcloud compute instance-groups list-instances \
-      "${name}" ${gcloud_region} --format=json | jq '.[1].instance')"
+      "${name}" ${gcloud_region} --format=json | jq '.[1].instance' | tr -d '"')"
     local previous_template="$(gcloud compute instances describe \
-      "${group_instance_name}" --format=json | \
+      ${group_instance_name} --format=json | \
```

Comment:
what if `group_instance_name` is something `--cli_arg foo` ?

Reply:
Since we are the only ones using this command whoever passes that as argument should probably use normal sounding instances :)

But I agree with @robertkiel on why now removing the quotes?

Reply:
Because bash will double quote it which breaks the call to gcloud entirely.